pacman/makepkg_mingw.conf

```diff
@@ -80,6 +80,16 @@ elif [[ "$MSYSTEM" == "CLANG32" ]]; then
   CFLAGS="-march=i686 -mtune=generic -O2 -pipe"
   CXXFLAGS="-march=i686 -mtune=generic -O2 -pipe"
   LDFLAGS="-pipe -Wl,--dynamicbase,--no-seh"
+elif [[ "$MSYSTEM" == "CLANGARM64" ]]; then
+  CARCH="aarch64"
+  CHOST="aarch64-w64-mingw32"
+  MINGW_CHOST="aarch64-w64-mingw32"
+  MINGW_PREFIX="/clangarm64"
+  MINGW_PACKAGE_PREFIX="mingw-w64-clang-aarch64"
+  CPPFLAGS="-D__USE_MINGW_ANSI_STDIO=1"
+  CFLAGS="-march=aarch64 -mtune=generic -O2 -pipe"
+  CXXFLAGS="-march=aarch64 -mtune=generic -O2 -pipe"
```

Comment:
Shouldn't this be `-march=armv8-a`?

Reply:
Possibly.  I just double-checked, and I have actually been running with no -march or -mtune.